Lib/http/cookies.py

```diff
@@ -519,21 +519,21 @@ def js_output(self, attrs=None):
             result.append(value.js_output(attrs))
         return _nulljoin(result)
 
-    def load(self, rawdata):
+    def load(self, rawdata, ignore_errors=False):
```

Comment:
Keyword-only argument is a much better choice for this nowadays.

Reply:
Done